### ISSUE TITLE
BUG: Save Results as Markdown Not Working

### ISSUE THREAD

**Issue by jeffcap1** [beekeeper-studio/beekeeper-studio#1622]
**Describe the bug**
Using the download Markdown option in the bottom right corner of the results panel does nothing when selected. Having Dev Tools open while trying this will reveal the following warning: `tabulator_esm.js:5143 Download Error - No such download type found:  md`

**To Reproduce**
1. Connect to a database and make a selection in a Query tab 
2. Click on the download button in the bottom right corner
3. Select Markdown
4. Nothing will happen

**Expected behavior**
A markdown file would be saved with the query results saved as a markdown table in the file.

**Screenshots / Gifs**
![image](https://github.com/beekeeper-studio/beekeeper-studio/assets/7104326/e62ef8eb-ec92-4129-a871-45417ba224a7)

**Version Information (please complete the following information):**
 - OS: MacOs 13.4.1
 - App Version: 3.9.17
 - Database type and version: 10.1.35-MariaDB

**Comment by not-night-but** [comment]
@jeffcap1 Yeah I just quickly took a look and was able to replicate this.

I looked into it a bit, and I think this was accidentally broken in #1494, but I think it should be an easy fix. So sorry about that!

Thanks for reporting this!
Day

**Comment by not-night-but** [comment]
@jeffcap1 fixed the issue! Just waiting on a review and then it should be good to merge into master. Not sure when we're pushing out the next release, but it'll be in that. In the meantime, once this PR is merged if you really need this feature and can't wait for the release, you are welcome to build from source. So sorry for any inconvenience!

Thanks again,
Day

**Comment by jeffcap1** [comment]
@not-night-but no problem happy it was easy to repro and fix! I worked around the issue by using the Markdown Copy feature and pasted it into a file.
